PubMed.js

```diff
@@ -93,8 +94,8 @@ function getUID(doc) {
 // retrieve itemprop elements for scraping books directly from page where UID is not available
 function getBookProps(doc) {
 	var main = doc.getElementById('maincontent');
-	if (!main) return;
-	
+	var booksections = doc.getElementById('book-sections');
+	if (!main && !booksections) return;
```

Comment:
I don't understand the logic here which you just changed recently. It is possible that `main` does not exist but `booksections` does, then this would continue (not both conditions fulfilled) and the next line would fail. I.e. `main` has to exist for the next line, but then we can change back to the original condition, because this will not change anything else. :thinking: